backends/qualcomm/passes/convert_addmm_back_to_linear.py

```diff
@@ -17,6 +20,10 @@
 
 
 class ConvertAddmmmmWithLinear(ExportPass):
+    """
+    Handle missing quantization tag for addmm op after decomposing
```

Comment:
are you using quantization_tag for identifying nodes pattern? If so addmm should already have that tag.

Reply:
I hope I didn’t confuse you. The quantization tag here refers to the quantization annotation.
https://github.com/CodeLinaro/executorch/blob/2dc87fe8352299e77701ab8e9019201f545a9a1b/backends/qualcomm/qnn_quantizer_utils.py#L25C25-L25C48